### DISCUSSION TITLE
Component renaming?

### DISCUSSION THREAD

**Discussion by andyburgessmd** [Budibase/budibase#12343] [Help] [answered]
Where has the ability to name component is builder design gone? If I add a new component and it is given a name like 'New Paragraph' how do I rename it to something meaningful?

**Comment by ConorWebb96** [comment, marked as answer]
Hey @andyburgessmd,

After selecting the component you wish to rename, click on its name on the right side of your screen. You should be able to free type and edit the name here. (Screenshot below to help with this explanation.)

<img width="306" alt="Screenshot 2023-11-09 at 09 39 55" src="https://github.com/Budibase/budibase/assets/126772285/91e54fb5-83ce-4b84-8dc8-ae90c125a745">

**Reply by andyburgessmd** [reply]
Thanks @ConorWebb96
